README.md

```diff
@@ -1,38 +1,47 @@
 [![Pub](https://img.shields.io/pub/v/flame.svg?style=popout)](https://pub.dartlang.org/packages/flame) ![Test](https://github.com/flame-engine/flame/workflows/Test/badge.svg?branch=master&event=push) [![Discord](https://img.shields.io/discord/509714518008528896.svg)](https://discord.gg/pxrBmy4)
 
-# :fire: flame
-
 <img src="https://i.imgur.com/vFDilXT.png" width="400">
 
 English | [简体中文](README-ZH.md)
 
-A minimalist Flutter game engine.
-
-Any help is appreciated! Comments, suggestions, issues, PRs! Give us a star to show your support!
+A minimalistic Flutter game engine.
 
 ## Help
 
-We have a Flame help channel on Fireslime's Discord, join it [here](https://discord.gg/pxrBmy4). Also we now have a [FAQ](FAQ.md), so please search for your question there first.
+We have a Flame help channel on Fireslime's Discord, join it [here](https://discord.gg/pxrBmy4). 
+
+We also have a [FAQ](FAQ.md), so please search for your question there first.
 
 ## Goals
 
 The goal of this project is to provide a complete set of out-of-the-way solutions for the common problems every game developed in Flutter will share.
 
-Currently it provides you with: a few utilities, images/sprites/sprite sheets, audio, a physics engine, a game loop and a component/object system.
+Currently it provides you with: 
+ - a game loop 
+ - bundles a physics engine (box2d)
+ - audio support
+ - effects and particles
+ - gesture and input support
+ - images, sprites and sprite sheets
+ - a component/object system
+ - basic Rive support
+ - and a few utilities to make development easier
```

Comment:
how about:

`a few [other] utilities`

Reply:
Or maybe just remove this line, it doesn't really add much?